org.jacoco.core.test.validation.java16/src/org/jacoco/core/test/validation/java16/targets/InstanceofTarget.java

```diff
@@ -20,7 +20,7 @@
 public class InstanceofTarget {
```

Comment:
Unfortunately GitHub doesn't allow me to comment on lines too far from changed ones 😞 

----

Javadoc mentions first preview ([JEP 305](https://openjdk.java.net/jeps/305)), but not second ([JEP 375](https://openjdk.java.net/jeps/375)) and not standardisation ([JEP 394](https://openjdk.java.net/jeps/394)). So I think will be nice to update it, especially if we stop testing this as preview feature. Maybe also convert it into a link?

```diff
 /**
- * This target exercises pattern matching for instanceof (JEP 305).
+ * This target exercises pattern matching for instanceof
+ * (<a href="https://openjdk.java.net/jeps/394">JEP 305</a>).
  */
```

As a remark - to me seems that reference to JEP is kinda more insightful than [reference to JLS](https://docs.oracle.com/javase/specs/jls/se16/html/jls-15.html#jls-15.20.2).

Reply:
Good idea, done.